lms/static/sass/partials/base/_variables.scss

```diff
@@ -205,14 +205,15 @@ $shadow-d1: rgba(0,0,0,0.4) !default;
 $shadow-d2: rgba($black, 0.6) !default;
 
 // system feedback-based colors
-$error-color: rgb(253, 87, 87) !default;
-$warning-color: rgb(181,42,103) !default;
+$error-color: rgb(203, 7, 18) !default;
```

Comment:
@alisan617 pinging again on this-- are there note pattern library colors that we can use?